buildpipeline/Core-Setup-Ubuntu14.04-x64.json

```diff
@@ -101,7 +101,7 @@
       "value": "PassedViaPipeBuild"
     },
     "CLI_NUGET_FEED_URL": {
-      "value": "https://www.myget.org/F/core-setup-testing"
+      "value": "https://dotnet.myget.org/F/cli-deps"
```

Comment:
Looking deeper into it, we are publishing it on "Ubuntu", version="14.04", which happens for both `x64` and `arm`.

We probably need a way to tell it only publish on `x64`.

I opened https://github.com/dotnet/core-setup/issues/1570 for this.